modules/util/source-formatter/src/main/resources/sourcechecks.xml

```diff
@@ -1734,7 +1734,7 @@
 		</check>
 		<check name="UpgradeSetResultsSetTotalMethodCheck">
 			<category name="Upgrade" />
-			<description name="Run code migration of method searchContainer.setResults to the searchContainer.setResultsAndTotal and delete searchContainer.setTotal" />
+			<description name="Run code migration of method searchContainer.setResults or setTotal to the searchContainer.setResultsAndTotal" />
```

Comment:
@TamyrisTorres, let's simplify:

`Replaces methods setResults and setTotal from SearchContainer with only the method setResultsAndTotal`